lib/gel/gemfile_parser.rb

```diff
@@ -65,9 +65,17 @@ def gem(name, *requirements, **options)
       @result.add_gem(name, requirements, options)
     end
 
-    def gemspec(path: ".", development_group: :development)
-      if file = Dir[File.join(File.expand_path(path, File.dirname(@result.filename)), "/*.gemspec")].first
-        spec = Gel::GemspecParser.parse(File.read(file), file)
+    def gemspec(name: nil, path: ".", development_group: :development)
+      dirname = File.expand_path(path, dirname)
```

Comment:
The `dirname` on the rhs here will always be nil... most of the time that's going to be equivalent, but I think for correctness we probably still want `File.dirname(@result.filename)` instead?

Reply:
Oops. Last minute "refactor" :man_facepalming: